code/modules/mob/mob_defines.dm

```diff
@@ -104,3 +104,5 @@
 	var/list/mousemove_intercept_objects
 
 	var/datum/click_intercept
+
+	var/throwforce = 10
```

Comment:
wrong way to do this. move throwforce from /obj to /atom/movable instead of an entirely new var

Reply:
this makes so much sense why am I a dork